### ISSUE TITLE
Logging updates: quiet mode and option to write log files

### ISSUE THREAD

**Issue by kevinschaper** [monarch-initiative/koza#82]
The logging output from Koza can get a bit noisy, and if ingests are running in parallel, it can be hard to keep the output straight. 

We should have a --quiet option that reduces logging that goes to the screen (maybe we still need ERROR level?). 

At the same time, we should add a --log option. I'm not sure if it should take a file name or a directory name, or even maybe whether it should just be a boolean flag and by default it will write `logs/{ingest_name}.log`

**Comment by glass-ships** [comment]
It looks like `--quiet` is already implemented in Koza, which is great! 

PR #84 aims to add a `--log` option and is in progress

**Comment by glass-ships** [comment]
Log option works now, will passively work on making it fancier. 

- [ ] Maybe if log is set, squash output to terminal and let write to file
- [ ] Format logger name into file lines
- [ ] Importing / passing on loggers to / from other modules? ??? @kevinschaper has thoughts on this

**Comment by glass-ships** [comment]
Should maybe make a new ticket for importing / passing on loggers to / from other modules? 
If it comes up anyway - it may not be an issue
